card/mod/05_standard/set/self/all.rb

```diff
@@ -20,3 +17,14 @@
     end
   end
 end
+
+
+module ClassMethods
+  def delete_old_sessions
+    if (months = Env.params[:months].to_i) && months > 0
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.